### PR TITLE
PHPCS Warnings and Error Fixes

### LINKED ISSUE #386: PHPCS Lint Errors
While working on my onboarding project, I discovered linting issues with wd_s and wanted to track them here. The fixes are listed here:

https://github.com/WebDevStudios/new-hire-haris-zulfiqar/commit/8583506b283015bd203919fa2f9f16dc9b11dc90

Apart from that, PHPCS throws a lint error for:

https://github.com/WebDevStudios/wd_s/blob/7bab27b394a66afe6c67af0f008562a4fe9b78c6/inc/scaffolding.php#L220

`Code after the BREAK statement on line 220 cannot be executed`

and

https://github.com/WebDevStudios/wd_s/blob/7bab27b394a66afe6c67af0f008562a4fe9b78c6/inc/acf.php#L184

`Processing form data without nonce verification.`

### PR DESCRIPTION
**Pull request by mharis**
Closes #386 

### DESCRIPTION ###
Passes PHPCS linting errors. Most were just alignment based with a few exceptions described in the comments directly on the code.

### OTHER ###
- [X] Is this issue accessible? (Section 508/WCAG 2.0AA)
- [X] Does this issue pass all the linting? (PHPCS, ESLint, SassLint)
- [X] Does this pass CBT?

### CODE REVIEW

File: functions.php [hunk 1]
```diff
@@ -105,7 +105,7 @@ function _s_widgets_init() {
 
 	// Define sidebars.
 	$sidebars = array(
-		'sidebar-1'  => esc_html__( 'Sidebar 1', '_s' ),
+		'sidebar-1' => esc_html__( 'Sidebar 1', '_s' ),
```
Comment (mharis) [inline functions.php:108]: Alignment fix

File: inc/acf.php [hunk 2]
```diff
@@ -155,9 +155,9 @@ function _s_has_block_expired( $args = array() ) {
 	$args = wp_parse_args( $args, $defaults );
 
 	// Get (Unix) times and convert to integer.
-	$now = (int) date( 'U' );
+	$now   = (int) date( 'U' );
```
Comment (mharis) [inline inc/acf.php:158]: Array alignment fix

File: inc/acf.php [hunk 3]
```diff
@@ -155,9 +155,9 @@ function _s_has_block_expired( $args = array() ) {
...
 	$start = (int) $args['start_date'];
-	$end = (int) $args['end_date'];
+	$end   = (int) $args['end_date'];
```
Comment (mharis) [inline inc/acf.php:160]: Array alignment fix

File: inc/acf.php [hunk 4]
```diff
@@ -181,7 +181,7 @@ function _s_has_block_expired( $args = array() ) {
 function _s_acf_admin_scripts() {
 
 	// If a SCRIPT_DEBUG constant is defined or there is a $_GET param of 'script_debug', load unminified files.
-	$suffix = ( defined( 'SCRIPT_DEBUG' ) && true === SCRIPT_DEBUG ) || isset( $_GET['script_debug'] ) ? '' : '.min';
+	$suffix = ( defined( 'SCRIPT_DEBUG' ) && true === SCRIPT_DEBUG ) || isset( $_GET['script_debug'] ) ? '' : '.min'; // phpcs:ignore
```
Comment (mharis) [inline inc/acf.php:184]: We don't need a nonce here. Ignored this line using PHPCS ignore comment.

File: inc/scaffolding.php [hunk 5]
```diff
@@ -105,7 +105,7 @@ function _s_scaffolding_allowed_html() {
 
 	// Add additional HTML tags to the wp_kses() allowed html filter.
 	$allowed_tags = array_merge( wp_kses_allowed_html( 'post' ), array(
-		'svg'   => array(
+		'svg'    => array(
```
Comment (mharis) [inline inc/scaffolding.php:108]: Array alignment fix

File: inc/scaffolding.php [hunk 6]
```diff
@@ -217,7 +217,10 @@ function _s_display_global_scaffolding_section( $args = array() ) {
 						<p><strong><?php echo esc_html( $font_var ); ?>:</strong> <span style="font-family: <?php echo esc_attr( $family ); ?>"><?php echo esc_html( $family ); ?></span></p>
 					<?php endforeach; ?>
 					</div>
-				<?php break; ?>
+				<?php
+					break;
+				default:
```
Comment (mharis) [inline inc/scaffolding.php:222]: All switches need a default break even if it's empty.

File: inc/scripts.php [hunk 7]
```diff
@@ -51,7 +51,7 @@ function _s_scripts() {
 	/**
 	 * If WP is in script debug, or we pass ?script_debug in a URL - set debug to true.
 	 */
-	$debug = ( defined( 'SCRIPT_DEBUG' ) && true === SCRIPT_DEBUG ) || ( isset( $_GET['script_debug'] ) ) ? true : false;
+	$debug = ( defined( 'SCRIPT_DEBUG' ) && true === SCRIPT_DEBUG ) || ( isset( $_GET['script_debug'] ) ) ? true : false; // phpcs:ignore
```
Comment (mharis) [inline inc/scripts.php:54]: We don't need a nonce here. Ignored this line using PHPCS ignore comment.

File: inc/scripts.php [hunk 8]
```diff
@@ -105,7 +105,7 @@ function _s_customizer_scripts() {
...
```
Comment (mharis) [inline inc/scripts.php:108]: We don't need a nonce here. Ignored this line using PHPCS ignore comment.

File: template-parts/scaffolding/scaffolding-globals.php [hunk 9]
```diff
@@ -27,9 +27,9 @@
 
 		// Theme fonts.
 		_s_display_global_scaffolding_section( array( // WPCS: XSS OK.
-			'global_type'  => 'fonts',
-			'title'        => 'Fonts',
-			'arguments'    => array(
+			'global_type' => 'fonts',
```
Comment (mharis) [inline template-parts/scaffolding/scaffolding-globals.php:30]: Array alignment.

**Review by gregrickaby — COMMENTED** [review]
@mharis This is great, thank you for tackling this. Which version of the code standards did you use?

**Comment by mharis** [comment]
`"webdevstudios/wds-coding-standards": "^1.1",`

**Review by gregrickaby — APPROVED** [review]
Looks good! Thanks! :thumbsup:

**Pull request merged**